akka-stream/src/main/scala/akka/stream/javadsl/SubSource.scala

```diff
@@ -1137,7 +1137,7 @@ class SubSource[+Out, +Mat](delegate: scaladsl.SubFlow[Out, Mat, scaladsl.Source
    *
    * '''Completes when''' upstream completes
    *
-   * '''Cancels when''' downstream cancels
+   * '''Cancels when''' downstream or Sink cancels
    */
   def alsoTo(that: Graph[SinkShape[Out], _]): SubSource[Out, Mat] =
```

Comment:
Could we have introduced an overload with the eager parameter?

Reply:
I think it was discussed and if you don't care about the side stream you should use the new wireTap (or graph dsl with Broadcast for power user full control)

Reply:
I'm not sure that's a good idea; that conflates the 2 decisions: 

- if the "other" should backpressure the upstream  (wireTap -- things get dropped for "other"; alsoTo -- things are backpressured)
- and what happens when "other" cancels (wireTap -- "other" completing does not cancel upstream; alsoTo -- completion of "other" does cancel the upstream)

But dropping on backpressure into "other" yet cancelling if it terminates could also be a valid use case?